pkg/manager/manager.go

```diff
@@ -302,6 +316,18 @@ func setOptionsDefaults(options Options) Options {
 	if options.newMetricsListener == nil {
 		options.newMetricsListener = metrics.NewListener
 	}
+	leaseDuration, renewDeadline, retryPeriod := defaultLeaseDuration, defaultRenewDeadline, defaultRetryPeriod
```

Comment:
We should pull the `options.newResourceLock = leaderelection.NewResourceLock` code fragment above closer to this block so that all leader election bits are in same place.